pkgs/build-support/setup-hooks/wrap-gapps-hook.sh

```diff
@@ -36,9 +36,12 @@ wrapGAppsHook() {
   done
 
   if [ -z "$dontWrapGApps" ]; then
-    for i in $prefix/bin/* $prefix/libexec/*; do
-      echo "Wrapping app $i"
-      wrapProgram "$i" "${gappsWrapperArgs[@]}"
+    # Loop over files, without falling prey to filenames containing
```

Comment:
Well, then maybe the comment is misleading since it suggests the change is intended to fix handling of file names that contain white space? Am I missing something?